GitUI/CommandsDialogs/SettingsDialog/Pages/ScriptsSettingsPage.cs

```diff
@@ -31,7 +31,7 @@ public partial class ScriptsSettingsPage : SettingsPageWithHeader
 Selected Commits:
 {sHashes}
 
-Selected Branch:
+On selected revision:
```

Comment:
"On ..." confuses me. It is rather "From ..." or "For ...".
The other headings should be adapted accordingly - at least the capitalization.
Thoughts?